drivers/avr/ws2812_i2c.c

```diff
@@ -21,11 +21,3 @@ void ws2812_setleds(LED_TYPE *ledarray, uint16_t leds) {
 
     i2c_transmit(WS2812_ADDRESS, (uint8_t *)ledarray, sizeof(LED_TYPE) * leds, WS2812_TIMEOUT);
 }
-
-// Setleds for SK6812RGBW
-void ws2812_setleds_rgbw(LED_TYPE *ledarray, uint16_t leds) {
-// not supported - for now error out if its enabled
-#ifdef RGBW
```

Comment:
This logic has to be maintained, as the ps2avrGB boards wont handle the extra bytes at all. Now the _rgbw function has been remove, the error logic should exist at the top of the file.